fluentforms/core/src/test/java/com/_4point/aem/fluentforms/impl/output/OutputServiceImplTest.java

```diff
@@ -497,24 +497,89 @@ void testGeneratePDFOutputBatch() {
 		fail("Not yet implemented");
 	}
 
-	@Disabled

```

Comment:
The generatePdfOutput and generatePrintedOutput are almost identical (except one takes a PdfOutputOptions argument and returns PDF while the other takes a PrintedOutputOptions argument and returned printer language).  Notice the number of tests for generatePdfOutput compared to the number of tests for generatePrintedOutput?  I was hoping for more generatePrintedOutput test cases to produce more code coverage.

Try right-clicking on OutputServiceImplTest and selecting Converage As/JUnit test, then go have a look at the OutputServiceImpl.java file.  You should see that there's a lot of green and very little red in the GeneratePdfArgumentBuilder but notice there's a lot of red in the GeneratePrintedOutputArgumentBuilder and no green.  This is a clear indication that the test coverage is lacking...

Would you mind adding more tests to get the test coverage up?  Take a look at the generatePdfOutput tests to see how they test the GeneratePdfOutputArgumentBuilder functionality.